Examples/CodePushDemoApp-pre0.49/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "CodePushDemoApp",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"start": "react-native start",
+		"test": "jest"
+	},
+	"dependencies": {
+		"react": "16.0.0-alpha.12",
+		"react-native": "^0.47.1",
```

Comment:
May be we should use latest pre 0.49 RN version here (i mean 0.48.4)?

Reply:
Don't see any differences within these two versions in comparison of configuration/usage, so not sure if this change is required

Reply:
@max-mironov okay, nevermind